src/controllers/quote.js

```diff
@@ -113,19 +113,13 @@ const InvalidAmountSpecifiedError = require('../errors/invalid-amount-specified-
 
 exports.get = function * () {
   validateAmounts(this.query.source_amount, this.query.destination_amount)
-  if (!this.config.features.quoteFullPath) {
-    this.body = yield model.getLocalQuote(this.query, this.ledgers, this.config)
-    return
+  if (!this.query.source_account && !this.query.source_ledger) {
+    throw new InvalidUriParameterError('Missing required parameter: source_ledger or source_account')
   }
-
-  if (!this.query.source_account) throw new InvalidUriParameterError('Missing required parameter: source_account')
-  if (!this.query.destination_account) throw new InvalidUriParameterError('Missing required parameter: destination_account')
-  this.body = yield model.getFullQuote(this.query, this.ledgers, this.config)
-}
-
-exports.getLocal = function * () {
-  validateAmounts(this.query.source_amount, this.query.destination_amount)
-  this.body = yield model.getLocalQuote(this.query, this.ledgers, this.config)
+  if (!this.query.destination_account && !this.query.destination_ledger) {
+    throw new InvalidUriParameterError('Missing required parameter: destination_ledger or destination_account')
+  }
+  this.body = yield model.getFullQuote(this.query, this.config)
```

Comment:
Since `src/models/quote.js` only exposes one method, it may be clearer to call this `getQuote`.